packages/react-components/src/AccessibilityScenarios/Slider.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+
+import { Slider } from '@fluentui/react-slider';
+import { Label } from '@fluentui/react-label';
+
+import { Scenario } from './utils';
+
+export const SoundControlSlidersAccessibilityScenario: React.FunctionComponent = () => {
+  return (
+    <Scenario pageTitle="Sound control sliders">
+      <h1>Sound control panel</h1>
```

Comment:
Can you make sure this is the appropriate heading level in the context of this page? If there's already an h1, this should be an h2.

Reply:
Corrected to &lt;h1&gt;.